### ISSUE TITLE
This vs findAndReplaceDOMText?

### ISSUE THREAD

**Issue by giorgio79** [nmartynenko/jquery-search-plugin#2]
Hello,

Been wondering how jquery search compares with https://github.com/padolsey/findAndReplaceDOMText

**Comment by nmartynenko** [comment]
Hi @giorgio79 

I'm not familiar with [padolsey/findAndReplaceDOMText](https://github.com/padolsey/findAndReplaceDOMText), but here is the list of features, which I can see:
- it finds and replace all concurrences, meanwhile my plugin can find and highlight only;
- highlighting in **findAndReplaceDOMText** plugin can be done with CSS (actually, replacing text with _span_ with specific CSS class), meanwhile this plugin can highlight with browsers capabilities
- **findAndReplaceDOMText** has no dependencies, mine does

Basically, my plugin just emulate `CTRL+F` capabilities with content boundaries (helpful for WEB-chats)+ capabilities of **findAndReplaceDOMText** plugin, though not that obvious.

Hope that provides some difference.
